packages/adapter-squid/src/constants/index.ts

```diff
@@ -1 +1,2 @@
 export * from "./ERC20Abi";
+export * from "./ErrorMessages";
```

Comment:
just an FYI that i learned recently: barrel files can slow down tooling (for larger repos at least)

https://marvinh.dev/blog/speeding-up-javascript-ecosystem-part-7/

Reply:
This is good to know. Will address this in a future PR. AFAIK the alternative would be to import directly from the required file. This would reduce that time but wouldn't look as clean 👎. If you have a suggestion for alternative approaches lmk.

Reply:
dunno tbh, personally also liked the cleanliness of barrel files but recently, I've started just kind of ignoring the first few import lines and just letting the tooling figure it out for me. vs code is pretty great in auto-importing and even considering file renames of imported modules, and eslint-plugin-import takes care of warning me about unused or otherwise invalid imports (think you can even configure it to sort imports alphabetically or however else you prefer)